README.md

```diff
@@ -1,5 +1,4 @@
 # <p align=center> Discord Guild Members Scraper
-## <p align=center> Check out my other project<br>[Darkend](https://darkend.tech)
```

Comment:
Preferably keep this innit? It's the author's choice to display what they want in the readme